app/js/components/orderEntry/addForm/AddForm.jsx

```diff
@@ -1,46 +1,110 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
 import FreeText from './FreeText';
 import StandardDose from './StandardDose';
+import DraftDataTable from './DraftDataTable';
 
-class AddForm extends React.Component {
+import { getOrderEntryConfigurations } from '../../../actions/orderEntryActions';
 
-  render() {
-    return (
-      <div className="row">
-        <div className="col-lg-11 col-sm-11 col-md-11 col-md-offset-1 col-sm-offset-1 col-lg-offset-1">
-          <ul className="nav nav-pills mb-3" id="pills-tab" role="tablist">
-            <li className="nav-item">
-              <a className="nav-link"
-                id="standard-tab"
-                data-toggle="pill"
-                href="#standard"
-                role="tab"
-                aria-controls="standard"
-                aria-selected="false">Standard Dosing
-              </a>
-            </li>
-            <li className="nav-item">
-              <a className="nav-link"
-                id="free-tab"
-                data-toggle="pill"
-                href="#free"
-                role="tab"
-                aria-controls="free"
-                aria-selected="false">Free Text
-              </a>
-            </li>
-          </ul>
-          <br />
-
-          <div className="tab-content" id="pills-tabContent">
-            <FreeText />
-            <StandardDose />
-          </div>
+export class AddForm extends React.Component {
+  state = {
+    drugFormPopulated: false,
+    action: '',
+    fields: {
+      completeInstructions: '',
+      dose: '',
+      dosingUnit: '',
+      frequency: '',
+      route: '',
+      duration: '',
+      durationUnit: '',
+      dispensingUnit: '',
+      dispensingQuantity: '',
+      reason: '',
+      note: '',
+    },
+  };
+  componentDidMount() {
+    this.props.getOrderEntryConfigurations();
+  }
+  handleCancel = () => {
+    // Cancel a draft drug order
```

Comment:
Is this an implementation to be handled as part of another task?

Reply:
Yes, since I divided my task into several sub-tasks.

Reply:
Alright.
Looks good